sources/views/tabletoolbar.js

```diff
@@ -0,0 +1,94 @@
+import {JetView} from "webix-jet";
+import {icons} from "../models/icons";
+
+export default class TableToolbar extends JetView {
+	constructor(app, name, data) {
+		super(app, name);
+		this._gridData = data;
+	}
+
+	config() {
+		const _ = this.app.getService("locale")._;
+		const datatable = {
+			view: "datatable",
+			localId: "settings:datatable",
+			scroll: "y",
+			select: true,
+			editable: true,
+			columns: [
+				{
+					id: "value",
```

Comment:
`Value`, as it is the main property. 
If you use `value` as an auxiliary property somewhere in templates/options, don't forget to update it as well ( use the `$change` or `$update` key in data scheme )

Reply:
fixed

Reply:
Not exactly. You create the field `value` for usage in options, which is correct
https://github.com/NataliMaevskaya/jet-test-app/blob/2b5568cd4e56322bcf944872a021a67cc884f8e4/sources/models/activityTypes.js#L7

but you don't update it after the `Value` was changed.
You need to improve the statements in `scheme`. You can either
- rename `$init` key to `$change` (as it will handle both adding and updating data)
- keep `$init` as it is and add a similar `$update` key.

The description of all methods you can find [here](https://docs.webix.com/desktop__data_scheme.html)

> _The same should be done for statuses_

Reply:
fixed